### PR TITLE
Feat 109  2 3 column views

### PR DESCRIPTION
**Pull request by bryanbraun**
# READY FOR REVIEW

## Summary
Add support for 2 and 3 column views by using Drupal's default "Grid" options, as shown below.

image

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Set up a view, using the "grid" format, test using 2 and 3 columns, and confirm that the content displays in the number of columns configured and scales down appropriately on smaller screen widths.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <h3>{{ title }}</h3>
+{% endif %}
+
+{% for row in items %}
+  {% for column in row.content %}
+      {% set viewItems = viewItems|merge([column.content]) %}
+  {% endfor %}
+{% endfor %}
+
+{{ pattern('grid', {items: viewItems, columns: options.columns}) }}
```
Comment (bryanbraun) [inline docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig:41]: @nrambeck, I see the `grid` UI pattern being used but not `grid_two_column` or `grid_three_column`. Are those UI patterns earlier iterations that can be deleted from this PR?

Comment (silverli) [reply docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig:41]: yes, @bryanbraun we started yesterday thinking the two and three column ui patterns could be used, but it looks like just the Grid one is used one.

Comment (silverli) [reply docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig:41]: I can make a commit with those changes

File: docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ *     - content: The row or column contents.
+ *
+ * @see template_preprocess_views_view_grid()
+ *
+ * @ingroup themeable
+ */
+#}
+{% set viewItems = [] %}
+{% if title %}
+  <h3>{{ title }}</h3>
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig:32]: should this be an h2?

**Review by silverli — CHANGES_REQUESTED** [review]
We need to remember to add the compiled css in when this PR is ready

**Comment by bryanbraun** [comment]
I've added the compiled CSS and fixed an issue where the column number setting wasn't getting passed to the "grid" UI pattern. As far as I can tell, it's all working now.

File: docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig (hunk 2 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/templates/views/views-view-grid.html.twig:32]: Bryan said core template uses h3 so this is fine

**Review by silverli — APPROVED** [review]
Looks great. Good catch on the column

**Pull request merged**